cppframework/Foo.cpp

```diff
@@ -2,9 +2,6 @@
 #include <string>
 #include <iostream>
 
-#define EXPORT __attribute__((visibility("default")))
```

Comment:
Nah. I'm testing CppInterface from app. And it is ObjC class. ObjC runtime sees it when I create it using NSClassFromString but not the linker.